src/netconfig.rs

```diff
@@ -63,6 +60,13 @@ fn handle_netconfig_values() -> Result<Option<Netconfig>, anyhow::Error> {
             );
         }
     }
+
+    if let Ok(gratuitous_arp) = dotenv::var("SEND_GRATUITOUS_ARP") {
+        if !gratuitous_arp.eq_ignore_ascii_case("auto") {
```

Comment:
I think ignoring the case here is wrong actually. From testing it quickly it seems that e.g. `AUTO` results in `<arp-notify>false</arp-notify>`

Reply:
https://github.com/openSUSE/wicked/blob/80b2e0da6d442e6a45e699323dfbe62d9fa17a21/client/suse/compat-suse.c#L6266

ack. looks like I mixed it up with values from e.g. [bootproto](https://github.com/openSUSE/wicked/blob/80b2e0da6d442e6a45e699323dfbe62d9fa17a21/client/suse/compat-suse.c#L6283) or others :-/